pennylane/ops/functions/iterative_qpe.py

```diff
@@ -39,34 +39,39 @@ def iterative_qpe(base, ancilla, iters):
 
     .. code-block:: python
 
-        dev = qml.device("default.qubit", shots = 5)
+        dev = qml.device("default.qubit", shots=5)
 
         @qml.qnode(dev)
         def circuit():
 
           # Initial state
-          qml.PauliX(wires = [0])
+          qml.PauliX(wires=[0])
 
           # Iterative QPE
-          measurements = qml.iterative_qpe(qml.RZ(2., wires = [0]), ancilla = 1, iters = 3)
+          measurements = qml.iterative_qpe(qml.RZ(2.0, wires=[0]), ancilla=1, iters=3)
 
-          return [qml.sample(op = meas) for meas in measurements]
+          return qml.sample(measurements)
 
     .. code-block:: pycon
 
         >>> print(circuit())
-        [array([0, 0, 0, 0, 0]), array([1, 0, 0, 0, 0]), array([0, 1, 1, 1, 1])]
+        [[0 0 1]
+         [0 0 1]
+         [0 0 1]
+         [1 1 1]
+         [0 0 1]]
 
-    The output is an array of size ``(number of iterations, number of shots)``.
+    The output is an array of size ``(number of shots, number of iterations)``.
```

Comment:
It was correct earlier because Guillermo was returning a list with `qml.sample`s for each iteration

Reply:
when did this change?

Reply:
This PR. It's just that Guillermo was returning `[qml.sample(m) for m in measurements]` before because we couldn't collect statistics on multiple mid-circuit measurements. Now that's possible, so I changed it to `qml.sample(measurements)` instead. In both cases, the shape is correct, but the original way was just because of constraints which no longer exist.

Reply:
oh I'm being dumb. you didn't change iqpe, but you did change the QNode in the docstring. lgtm